src/goreleaser.ts

```diff
@@ -129,6 +129,18 @@ const completionSpec: Fig.Spec = {
         },
       ],
     },
+    {
+      name: ["hc", "healthcheck"],
+      description: "Checks if needed tools are installed",
+      options: [
+        {
+          name: ["--config", "-f"],
+          description: "Configuration file",
+          args: { name: "config" },
```

Comment:
@caarlos0 this can be generated automatically so the cobra definitions are likely missing some kind of property
```suggestion
          args: { name: "config", template: "filepaths" },
```